### DISCUSSION TITLE
Increase the size of User.ID field (currently 32-char) to 64-char

### DISCUSSION THREAD

**Discussion by josenogueiradalmeida** [openkfw/TruBudget#861] [Ideas]
During our tests, we noticed that we can not fill the User ID with one of our client's email (monique.bendahan@fas-amazonas.org). 

For this reason, we kindly ask to increase the size of the field to support 64 characters.

Thanks in advance.

**Comment by josenogueiradalmeida** [comment]
Another important example of this limitation: trubudget-noreply@bndes.gov.br almost blows the 32-char limit.

**Comment by mayrmartin** [comment]
Thank you for your feature request! It will be addressed in #887.
